scalyr_agent/monitor_utils/k8s.py

```diff
@@ -2122,16 +2126,22 @@ class KubeletApi(object):
         A class for querying the kubelet API
     """
 
-    def __init__(self, k8s, port=10255, host_ip=None):
+    def __init__(
+        self,
+        k8s,
+        host_ip=None,
+        kubelet_url_template=Template("https://${host_ip}:10250"),
```

Comment:
Let's add this as a top-level configuration option in `configuration.py`.

Reply:
Since `KubeletAPI` doesn't currently have the config, would it be better to pass in the config, or to pass in the already constructed template?